src/modules/fw_pos_control_l1/fw_pos_control_l1_main.cpp

```diff
@@ -988,24 +1000,29 @@ bool FixedwingPositionControl::update_desired_altitude(float dt)
 		_althold_epv = _global_pos.epv;
 	}
 
-	// XXX the sign magic in this function needs to be fixed
-
+	/*
+	 * Manual control has as convention the rotation around
+	 * an axis. Positive X means to rotate positively around
+	 * the X axis in NED frame, which is pitching down
```

Comment:
Y axis in the NED frame
manual control x --> - y in NED
see https://github.com/PX4/Firmware/blob/master/msg/manual_control_setpoint.msg#L16